src/kontrol/kdist/foundry.md

```diff
@@ -1074,7 +1001,7 @@ If the pair is not present in the whitelist `WLIST` then `KEVM` goes into an err
 function allowCallsToAddress(address) external;
 ```
 
-Adds an account address to the whitelist. The execution of the modified KEVM will stop when a call has been made to an address which is not in the whitelist.
+Adds an account address to the whitelist. The execution of the modified Kontrol will stop when a call has been made to an address which is not in the whitelist.
```

Comment:
Same as above. Maybe you could rewrite as `.. of the modified EVM semantics ..`?

Reply:
I changed it back to KEVM, same as above. I wasn't sure which KEVMs needed to be changed to Kontrol. SO I really appreciate the review! Thanks so much!